graphistry/feature_utils.py

```diff
@@ -228,7 +228,7 @@ def resolve_feature_engine(
 YSymbolic = Optional[Union[List[str], str, pd.DataFrame]]
 
 
-def resolve_y(df: Optional[pd.DataFrame], y: YSymbolic) -> pd.DataFrame:
+def resolve_y(df: Optional[pd.DataFrame], y: YSymbolic, cudf: None) -> pd.DataFrame:
```

Comment:
can we keep `cudf` out of the type sig here?

Reply:
sure that makes sense, i was trying to minimize calling `lazy_cuda` import